ee/desktop/runner/server/server.go

```diff
@@ -0,0 +1,144 @@
+package server
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"net/http"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/kit/ulid"
+)
+
+type RunnerServer struct {
```

Comment:
Mind adding a short comment about what the runner server's responsibilities are? `Allows user desktop processes to monitor + notify parent process` or similar

Reply:
done